Buildings/package.mo

```diff
@@ -341,6 +341,16 @@ that can lead to wrong simulation results):
                        <a href=\"https://github.com/lbl-srg/modelica-buildings/issues/3597\">Buildings, #3597</a>.
     </td>
 </tr>
+<tr><td colspan=\"2\"><b>Buildings.Airflow.Multizone</b>
+    </td>
+</tr>
+<tr><td valign=\"top\">Buildings.Airflow.Multizone.BaseClasses.interpolate
+    </td>
+    <td valign=\"top\">Corrected implementation to ensure that the function is first-order smooth.<br/>
```

Comment:
Please replace "first-order smooth" with "once continuously differentiable".

Reply:
Corrected in b8313ff4efdabaff9f9af48a1d75c56a0d79b6ee.